wled00/overlay.cpp

```diff
@@ -11,6 +11,7 @@ void _overlayAnalogClock()
   {
     _overlayAnalogCountdown(); return;
   }
+  uint8_t brightness = strip.getBrightness();
```

Comment:
@blazoncek not sure what is better to retrieve the global brightness - simply use global `bri`, or use `‎WS2812FX::getBrightness‎()`?

Reply:
Yes. That would be in line with other code that uses it.

Reply:
ok